dbt/include/firebolt/macros/materializations/table.sql

```diff
@@ -2,21 +2,19 @@
 {# Note that a nearly identical materialization appears
    in table.sql #}
   {%- set identifier = model['alias'] -%} {# alias comes from where? #}
-  {%- set old_relation = adapter.get_relation(database=database,
-                                              schema=schema,
-                                              identifier=identifier) -%}
   {%- set target_relation = api.Relation.create(database=database,
                                                 schema=schema,
                                                 identifier=identifier,
                                                 type='table') -%}
-  {%- set exists_as_table = (old_relation is not none and old_relation.is_table) -%}
-  {%- set exists_as_view = (old_relation is not none and old_relation.is_view) -%}
+  {%- set target_relation_view = api.Relation.create(database=database,
+                                                schema=schema,
+                                                identifier=identifier,
+                                                type='view') -%}
```

Comment:
```suggestion
  {%- set target_relation_view = api.Relation.create(database=database,
                                                    schema=schema,
                                                    identifier=identifier,
                                                    type='view') -%}
```

Reply:
Iirc, per PEP8, to have a four-space dedent none of the arguments can appear on the first line. I.e. `database` would need to move to the next line.

Mostly, though, it just didn't look good _to me_. 😉